example/README.md

```diff
@@ -0,0 +1,342 @@
+    export App, echo
+
+    func echo(msg: string) -> string:
+        services <- App.services()
+
+        on services.echoService.default[0].peerId:
+            EchoService services.echoService.default[0].serviceId
+            res <- EchoService.echo(msg)
+        <- res
+    ```
+
+3. Let’s test our echo service:
+
+    ```bash
+     fluence run -f 'echo("hi")'
+    ```
+
+    Output:
+
+    ```bash
+    $ fluence run -f 'echo("hi")'
+    Running:
+      function: echo("hi")
+      relay: /dns4/kras-08.fluence.dev/tcp/19001/wss/p2p/12D3KooWFtf3rfCDAfWwt6oLZYZbDfn9Vn7bv7g6QjjQxUUEFVBt
+    ... done
+
+    Result:
+
+    "12D3KooWCMr9mU894i8JXAFqpgoFtx6qnV1LFPSfVc3Y34N4h4LS: hi"
+    ```
+
+
+## Run echo service in JS/TS:
+
+1. Install dependencies:
+
+    ```bash
+    npm i
+    ```
+
+2. Check out [src/aqua/export.aqua](src/aqua/export.aqua) with an export of EchoService API to JS/TS:
+
+    ```bash
+    module Export
+    import "services/echoService.aqua"
+    export EchoService
+    ```
+
+3. The following [code](src/echo.ts) registers local EchoService:
+
+    ```tsx
+    // register local service with service id "echo"
+    await registerEchoService(serviceId, {echo: (msg) => {
+        console.log("Received message:", msg);
+        return peerId + ": " + msg
+    }});
+    ```
+
+4. The following [code](src/aqua/main.aqua) is calling our local JS/TS peer with `EchoService`:
+
+    ```rust
+    func echoJS(peer: string, relay: string, serviceId: string, msg: string) -> string:
+        on peer via relay:
+            EchoService serviceId
+            res <- EchoService.echo(msg)
+        <- res
+    ```
+    <a id="running-service"></a>
+5. Start client with service: `npm run start`
+    ```bash
+    ...
+    > example@1.0.0 start
+    > node src/echo.js
+
+    📗 created a fluence peer 12D3KooWCmnhnGvKTqEXpVLzdrYu3TkQ3HcLyArGJpLPooJQ69dN with relay 12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi
+    Copy this code:
+    fluence run -f 'echoJS("12D3KooWCmnhnGvKTqEXpVLzdrYu3TkQ3HcLyArGJpLPooJQ69dN", "12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi", "echo", "msg")'
+    ```
+
+6. Open a new terminal and copy the last line and execute to check echo service:
+
+    ```bash
+    $ fluence run -f 'echoJS("12D3KooWCmnhnGvKTqEXpVLzdrYu3TkQ3HcLyArGJpLPooJQ69dN", "12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi", "echo", "msg")'
+    Running:
+      function: echoJS("12D3KooWCmnhnGvKTqEXpVLzdrYu3TkQ3HcLyArGJpLPooJQ69dN", "12D3KooWFEwNWcHqi9rtsmDhsYcDbRUCDXH84RC4FW6UfsFWaoHi", "echo", "msg")
+      relay: /dns4/kras-09.fluence.dev/tcp/19001/wss/p2p/12D3KooWD7CvsYcpF9HE9CCV9aY3SJ317tkXVykjtZnht2EbzDPm
+    ... done
+
+    Result:
+
+    "12D3KooWCmnhnGvKTqEXpVLzdrYu3TkQ3HcLyArGJpLPooJQ69dN: msg"
+    ```
+
+
+## Register both services in Registry
+
+If we, as service providers, want to be all echo services discoverable without specifying exact peer and service ids, we should use Registry to **advertise** our services.
```

Comment:
I'd rephrase it: 

As service providers we would like all our echo services to be discoverable without specifying particular peer and service ids, in order to achieve that we should use Registry to **advertise** our services.